include/swift/ClangImporter/CXXMethodBridging.h

```diff
@@ -3,75 +3,78 @@
 
 #include "swift/AST/Decl.h"
 #include "clang/AST/DeclCXX.h"
+#include "llvm/ADT/STLExtras.h"
```

Comment:
@zoecarver should hopefully pass linux ci now, imported `llvm/ADT/STLExtras.h` to expose `llvm::any_of`

Reply:
@cabmeurer Nope :(

Reply:
@xwu  Super weird, any idea why this would only fail on Linux?

Reply:
I can obviously use a loop as a workaround, but `llvm::any_of` i used everywhere. Could this be an issue with the pipeline?

Reply:
I wish I could be of more help here. Wonder if it's a red herring and the actual problem lies somewhere nearby?

Reply:
No worries! Will you run CI for me again when you get a chance?